packages/react-components/lib/trashbin-control-button-group-reducer.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+
+export type TrashBinState<T> = {
+  current: T;
+  trash: T;
```

Comment:
I was afraid that when we empty the `trash` state triggers a component re-render, that's why I was using the ref, but it seems not to be the case. So, good solution =)